Source/Core/ClippingPlaneCollection.js

```diff
@@ -574,27 +570,62 @@ define([
      */
     ClippingPlaneCollection.TEXTURE_WIDTH = CesiumMath.nextPowerOfTwo(Math.ceil(Math.sqrt(ClippingPlaneCollection.MAX_CLIPPING_PLANES * 2)));
 
+    /**
+     * Returns true if this object was destroyed; otherwise, false.
+     * <br /><br />
+     * If this object was destroyed, it should not be used; calling any function other than
+     * <code>isDestroyed</code> will result in a {@link DeveloperError} exception.
+     *
+     * @returns {Boolean} <code>true</code> if this object was destroyed; otherwise, <code>false</code>.
+     *
+     * @see ClippingPlaneCollection#destroy
+     */
     ClippingPlaneCollection.prototype.isDestroyed = function() {
         return false;
     };
 
     /**
-     * Destroys this ClippingPlaneCollection if it either has no owner or a reference to the owner is passed in.
-     * We only expect an owner to be defined if this collection is shared by one Cesium3DTileset
-     * and a series of Cesium3DTiles with Model content.
+     * Destroys this ClippingPlaneCollectionand its WebGL resources.
      * @private
+     */
+    ClippingPlaneCollection.prototype.destroy = function() {
```

Comment:
Round out the doc for this. For example:

```
    /**
     * Destroys the WebGL resources held by this object.  Destroying an object allows for deterministic
     * release of WebGL resources, instead of relying on the garbage collector to destroy this object.
     * <br /><br />
     * Once an object is destroyed, it should not be used; calling any function other than
     * <code>isDestroyed</code> will result in a {@link DeveloperError} exception.  Therefore,
     * assign the return value (<code>undefined</code>) to the object as done in the example.
     *
     * @returns {undefined}
     *
     * @exception {DeveloperError} This object was destroyed, i.e., destroy() was called.
     *
     *
     * @example
     * clippingPlanes = clippingPlanes && clippingPlanes .destroy();
     *
     * @see ClippingPlaneCollection#isDestroyed
     */
```

Reply:
Also remove `@private`. See https://github.com/AnalyticalGraphicsInc/cesium/pull/5615#discussion_r166123282